server/services/store/sqlstore/migrations/000018_populate_categories.up.sql

```diff
@@ -6,7 +6,7 @@ CREATE TABLE {{.prefix}}categories (
     user_id varchar(32) NOT NULL,
     team_id varchar(32) NOT NULL,
     {{if not .sqlite}}
-        channel_id varchar(32) NOT NULL,
+      channel_id varchar(32) NOT NULL,
```

Comment:
Does anyone know why channel_id is in the categories table? It doesn't appear to be used anywhere.